tests/e2e/tests/content-releases/releases-page.spec.ts

```diff
@@ -43,57 +43,65 @@ describeOnCondition(edition === 'EE')('Releases page', () => {
 
-      // Create a release
-      const newReleaseName = 'The Diamond Dogs';
-      await page.getByRole('textbox', { name: 'Name' }).fill(newReleaseName);
+    // Create a release
+    const newReleaseName = 'The Diamond Dogs';
+    await page.getByRole('textbox', { name: 'Name' }).fill(newReleaseName);
 
-      // Select valid date and time
-      await page
-        .getByRole('combobox', {
-          name: 'Date',
-        })
-        .click();
+    // Select valid date and time
+    await page
+      .getByRole('combobox', {
+        name: 'Date',
+      })
+      .click();
+
+    const date = new Date();
+    date.setDate(date.getDate() + 1);
+    const formattedDate = date.toLocaleDateString('en-US', {
+      weekday: 'long',
+      month: 'long',
+      day: 'numeric',
+      year: 'numeric',
+    });
 
-      const date = new Date();
-      date.setDate(date.getDate() + 1);
-      const formattedDate = date.toLocaleDateString('en-US', {
-        weekday: 'long',
-        month: 'long',
-        day: 'numeric',
-        year: 'numeric',
-      });
+    await page.getByLabel(formattedDate).click();
 
-      await page.getByLabel(formattedDate).click();
+    await page.getByRole('combobox', { name: /^time$/i }).click();
 
-      await page
-        .getByRole('combobox', {
-          name: /^time$/i,
-        })
-        .click();
+    const optionIsVisible = await page.getByRole('option', { name: '08:00' }).isVisible();
+    console.log('08:00 option visible:', optionIsVisible);
 
-      await page.getByRole('option', { name: '14:00' }).click();
-      await page.getByRole('button', { name: 'Continue' }).click();
-      // Wait for client side redirect to created release
-      await page.waitForURL('/admin/plugins/content-releases/*');
-      await expect(page.getByRole('heading', { name: newReleaseName })).toBeVisible();
+    const buttonVisible = await page.getByRole('button', { name: 'Continue' }).isVisible();
+    console.log('Continue button visible:', buttonVisible);
 
-      // Navigate back to the release page to see the newly created release
-      await page.getByRole('link', { name: 'Releases' }).click();
-      await expect(page.getByRole('link', { name: `${newReleaseName}` })).toBeVisible();
-    }
-  );
+    // Log URL changes
+    page.on('framenavigated', async (frame) => {
+      if (frame === page.mainFrame()) {
+        console.log('Navigation occurred to:', frame.url());
+      }
+    });
+    await page.getByRole('option', { name: '08:00' }).click();
+    await page.getByRole('button', { name: 'Continue' }).click();
+
+    // Wait for client side redirect to created release
+    await page.waitForURL('/admin/plugins/content-releases/*');
+    await expect(page.getByRole('heading', { name: newReleaseName })).toBeVisible();
+
+    // Navigate back to the release page to see the newly created release
+    await page.getByRole('link', { name: 'Releases' }).click();
```

Comment:
Looks like a good spot for clickAndWait (I know await toBeVisible below should make this redundant, but I have flakiness ptsd 🙃)